easybuild/easyconfigs/n/NiBabel/NiBabel-3.2.1-foss-2021a.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'PythonBundle'
+
+name = 'NiBabel'
+version = '3.2.1'
+
+homepage = 'https://nipy.github.io/nibabel'
+description = """NiBabel provides read/write access to some common medical and neuroimaging file formats,
+ including: ANALYZE (plain, SPM99, SPM2 and later), GIFTI, NIfTI1, NIfTI2, MINC1, MINC2, MGH and ECAT
+ as well as Philips PAR/REC. We can read and write Freesurfer geometry, and read Freesurfer morphometry and
+ annotation files. There is some very limited support for DICOM. NiBabel is the successor of PyNIfTI."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('CUDA', '11.3.1', '', True),
```

Comment:
This dependency can be removed. NiBabel does not use/need CUDA.

Reply:
How did that sneak in there? ;) Thank you for the feedback. I have removed it!